src/governance/vote.h

```diff
@@ -18,24 +18,25 @@ class CKeyID;
 class PeerManager;
 
 // INTENTION OF MASTERNODES REGARDING ITEM
-enum vote_outcome_enum_t : uint8_t {
-    VOTE_OUTCOME_NONE      = 0,
-    VOTE_OUTCOME_YES       = 1,
-    VOTE_OUTCOME_NO        = 2,
-    VOTE_OUTCOME_ABSTAIN   = 3
+enum vote_outcome_enum_t : int {
```

Comment:
From [N4659 C++ 7.2/5](https://timsong-cpp.github.io/cppwp/n4659/dcl.enum#7):

>    For an enumeration whose underlying type is not fixed, the underlying type is an integral type that can represent all the enumerator values defined in the enumeration. If no integral type can represent all the enumerator values, the enumeration is ill-formed. It is implementation-defined which integral type is used as the underlying type except that the underlying type shall not be larger than int unless the value of an enumerator cannot fit in an int or unsigned int. If the enumerator-list is empty, the underlying type is as if the enumeration had a single enumerator with value 0.

nit: type can be omitted here completely:
```
-enum vote_outcome_enum_t : int {
+enum vote_outcome_enum_t {
```